iogt/templates/account/login.html

```diff
@@ -18,14 +18,13 @@ <h1>{% trans "Log In" %}</h1>
         </div>
         <div class="profile-form__btns">
             <button type="submit"
-                    class="cust-btn cust-btn--dark profile-form__btn"><span>Log in</span>
+                    class="cust-btn cust-btn--dark profile-form__btn">
+                <span>{% trans "Log in" %}</span>
             </button>
             <a href="{% url 'account_signup' %}"
-               class="cust-btn profile-form__btn"><span>Create your account</span></a>
+               class="cust-btn profile-form__btn"><span>{% trans "Create your account" %}</span></a>
         </div>
-        <p class="text profile-form__desc">A quia quas et atque eos vero
-            temporibus quis ullam. Recusandae recusandae omnis aspernatur
-            quisquam at dolorem aspernatur quod non.</p>
+        <p class="text profile-form__desc">{% trans "A quia quas et atque eos vero temporibus quis ullam. Recusandae recusandae omnis aspernatur quisquam at dolorem aspernatur quod non."%}</p>
```

Comment:
What should this placeholder text eventually be?
Should this get a [noop tag](https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#translate-template-tag) until the real text is in? Same in signup.html